.github/workflows/coverage.yml

```diff
@@ -1,24 +1,28 @@
-name: nim-libp2p codecov builds
+name: Coverage
 
 on:
-  #On push to common branches, this computes the "bases stats" for PRs
+  # On push to common branches, this computes the coverage PRs will compared against
   push:
     branches:
       - master
   pull_request:
+    branches:
```

Comment:
I don't believe this is necessary.

Reply:
As I answered to a comment you made while this PR was in draft, the `Upload coverage to codecov` request has a monthly limit as we're not using a paid tier. Thus, this two lines attempt to reduce the number of calls to `codecov` in order not to exceed that limit. 

I do agree, though, that maybe I'm providing a solution a (yet) non-existing problem, as we probably still haven't reached that limit. Also, the only affected step is `Upload coverage to codecov`, so running that job conditionally might be enough, as we probably don't need all coverages uploaded to codecov, but the `master`-targeted PR ones.

What do you think about the second approach?

Reply:
We don't need to spend time doing something if there's no evidence it's an issue.

Reply:
Right, as long as you remember to fix that when it happens.

Reply:
Right, as long as you remember to fix that when it happens I'm fine with that.